components/Collections/src/MDCCollectionViewController.m

```diff
@@ -311,10 +334,10 @@ - (UICollectionReusableView *)collectionView:(UICollectionView *)collectionView
   if ([kind isEqualToString:MDCCollectionInfoBarKindHeader] ||
       [kind isEqualToString:MDCCollectionInfoBarKindFooter]) {
     NSString *identifier = NSStringFromClass([MDCCollectionInfoBarView class]);
-    identifier = [identifier stringByAppendingFormat:@".%@", kind];
-    [collectionView registerClass:[MDCCollectionInfoBarView class]
-        forSupplementaryViewOfKind:kind
-               withReuseIdentifier:identifier];
+    // KM: Simplify identifier
```

Comment:
(The NSLogs will be useful for testing, so I'll keep them until everyone is satisfied)